src/main/java/step2/view/ResultView.java

```diff
@@ -0,0 +1,33 @@
+package step2.view;
+
+import step2.domain.Prize;
+import step2.domain.LottoTicket;
+import step2.domain.LottoTickets;
+import step2.utils.StringUtil;
+
+import java.util.List;
+import java.util.Map;
+
+public class ResultView {
+    public static void showMyLottoNumbers(LottoTickets lottoTickets){
+        List<LottoTicket> lottoList = lottoTickets.getLottoTickets();
+        System.out.println(lottoTickets.getTheNumberOfLottoTickets()+"장을 구매했습니다.");
+        for (LottoTicket lottoTicket : lottoList) {
+            List<Integer> lottoNumbers = lottoTicket.getLottoNumbers();
+            System.out.println(StringUtil.splitStringFromList(lottoNumbers));
+        }
+        System.out.println();
+    }
+
+    public static void showResult(Map<Integer, Integer> result){
+        System.out.println("당첨 통계\n---------");
```

Comment:
```suggestion
        System.out.printf("당첨 통계%n---------%n");
```
`\n` 은 시스템 의존적인 줄바꿈 문자 입니다. 현제 OS 에 맞는 줄바꿈은 `System.lineSeparator()` 또는 `String.format` 이나 `System.out.printf` 에서 `%n` 문자를 사용하는 방법이 있습니다.